### PR TITLE
[FIX] Correct reqs for validators & info about deposits and bonds

### PR DESCRIPTION
**Pull request by havan**
* Updated and fixed requirements in validator document to clarify locked/unlocked and deposits/bonds.
* Placeholder for deposits and bonds page
* Some fixes to clarify things in validator FAQ

### CODE REVIEW

**Comment by havan** [comment]
Deployed for easier review [here](https://playground.docs.camino.network/guides/deposits-and-bonds/index.html).

File: docs/faq/validator.md [hunk 1]
```diff
@@ -5,47 +5,56 @@ description: Frequently asked questions about validation on the camino and colum
...
 
-## Can I add more Camino (CAM) tokens to my validator while the staking period is in progress?
+### What are staking parameters for validators on Camino Network?
+
+- The exact amount that a validator must stake is **100,000 CAM**.
+- The time period one can stake funds for a validator is between **2 weeks** (min) and **1 year** (max).
+
+:::note
+
+When a validator completes its role in validating the Camino Mainnet, they will receive their staked Camino tokens back. Additionally, they may also receive a reward for their contribution in securing the network. However, it's worth noting that a validator will only receive a validation reward if it has been sufficiently responsive and accurate during the validation period.
```
Comment (evlekht) [inline docs/faq/validator.md:29]: Thing about reward seems wrong. Our validators receives rewards during their validation period, not at the end of it. And those rewards must be also claimed via tx, they'r not just minted for validators automatically.

File: docs/guides/add-validator-with-curl.md [hunk 2]
```diff
@@ -42,8 +43,8 @@ Further below, you will find instructions about how to get these information by
   Keep it safe and secure!
 - **Wallet address & private key**: You will need your wallet address and private key.<br/>
   Remember to keep your private key safe!
-- **100,000 unlocked CAMs** in your P-Chain wallet.<br/>
-  You can check this on the online [wallet](https://wallet.camino.network) to ensure that you have at least 100,000 unlocked tokens on the P-Chain.
+- **100,000 unbonded CAMs** in your P-Chain wallet. And a few CAMs for paying for transaction fees.<br/>
```
Comment (evlekht) [inline docs/guides/add-validator-with-curl.md:46]: I'd say not-bonded, but may be I'm wrong :)

File: docs/guides/add-validator-with-curl.md [hunk 3]
```diff
@@ -54,6 +55,22 @@ Pre-sale participants have already been designated as Consortium Members on the
 
 :::
 
+:::info UNBONDED CAM TOKENS
+
+In the Camino Network, CAM tokens can be utilized as securities for various purposes such as deposit rewards, validator stakes, or DAO proposals. There are two types of these securities: **deposits** and **bonds**.
+
+Individuals can deposit CAM tokens for a certain period of time to earn rewards. In this scenario, the tokens are "**deposited**" and their state is "**locked**".
```
Comment (evlekht) [inline docs/guides/add-validator-with-curl.md:62]: What does it mean exactly, that their state is **locked**?

File: docs/faq/validator.md (hunk 1 above)
Comment (havan) [reply docs/faq/validator.md:29]: I think I'll remote that part completely, that part is old and confusing.

File: docs/guides/add-validator-with-curl.md (hunk 2 above)
Comment (havan) [reply docs/guides/add-validator-with-curl.md]: I've checked a few dictionaries and settled for **non-bonded** :)

File: docs/guides/add-validator-with-curl.md (hunk 3 above)
Comment (havan) [reply docs/guides/add-validator-with-curl.md]: Just trying to say that these tokens would be locked. Remote "state" part, so it's not confusing.

**Pull request merged**